src/Motor.Extensions.Diagnostics.Metrics/PrometheusDelegatingMessageHandler.cs

```diff
@@ -10,20 +10,19 @@ namespace Motor.Extensions.Diagnostics.Metrics;
 public class PrometheusDelegatingMessageHandler<TInput> : DelegatingMessageHandler<TInput>
     where TInput : class
 {
-    private readonly IMetricFamily<ICounter> _messageProcessingTotal;
+    private readonly IMetricFamily<ISummary> _messageProcessing;
 
-    public PrometheusDelegatingMessageHandler(
-        IMetricsFactory<PrometheusDelegatingMessageHandler<TInput>> metricsFactory)
+    public PrometheusDelegatingMessageHandler(IMetricsFactory<PrometheusDelegatingMessageHandler<TInput>> metricsFactory)
     {
-        _messageProcessingTotal =
-            metricsFactory.CreateCounter("message_processing_total", "Message processing status total", false, "status");
```

Comment:
Note: We should make sure to put this breaking change into the release notes, when releasing this. Someone might be using this metric.